app/scripts/controllers/form.js

```diff
@@ -13,22 +13,16 @@ angular.module('ludwig').controller('FormCtrl', function($scope, $http, $state,
     };
 
     $scope.test.expectedResults.forEach(function(expectedResult) {
-        expectedResult.ref = _.find($scope.possibleValues, { id: expectedResult.code });
```

Comment:
I moved it [here](https://github.com/sgmap/ludwig-ui/blob/xp-adpa-93/app/scripts/services/acceptanceTest.js#L58), as I needed to have this property for tests in all contexts and not just on this page.